src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -83,15 +104,16 @@ function BoxAISidebarContent(props: ApiWrapperProps) {
                     {formatMessage(messages.sidebarBoxAITitle)}
                 </Text>
                 {isAIStudioAgentSelectorEnabled && (
-                    <BoxAiAgentSelectorWithApi
-                        fetcher={getAIStudioAgents}
-                        hostAppName={hostAppName}
-                        onSelectAgent={onSelectAgent}
-                        recordAction={recordAction}
-                        shouldHideAgentSelectorOnLoad
-                        // @ts-ignore variant will be available in higher version
-                        variant="sidebar"
-                    />
+                    <div className="bcs-BoxAISidebar-agent-selector-container">
```

Comment:
so for suit naming, the class name should start with the component name and then descendants are listed in camelcase after a dash: `bcs-BoxAISidebarContent-agentSelector`

but since `bcs-BoxAISidebar` is already the pattern in this file, I guess call this `bcs-BoxAISidebar-agentSelector`?